style.css

```diff
@@ -1,50 +1,178 @@
-/** {
+* {
 	box-sizing:border-box;
-	border: 1px solid red;
-}*/
+/*	border: 1px solid red;
+*/}
+	
+html {
+	font-size:14px;
+}
 
-html {font-size:14px;),
 
+body {
+
+	background-image: url("img/black.png");
+	background-attachment: fixed;
+	color:white	
 }
 
-body {
-	background-image:url('img/wheat.png');
-	size:cover;
-	background-repeat:repeat;
+
+.fullscreen-bg__video {
+    position: absolute;
+    width: 100%;
+    height: 900px;
 }
 
 .container {
+	width:60%;
+	margin:0 auto;
 	overflow:auto;
+	text-align:center;
 }
 
-div {
-	color:white;
+a {
+	text-decoration:none;
+	color:inherit;
+}
+
+a:hover {
+	transition:ease 0.5s;
+	color: #FBB025;
+	border-top:solid 2px #FC630E;
+	padding-top:20px;
+	border-radius:0px;
+	overflow:auto;
+}
+
+nav {
+	overflow:hidden;
+	position:fixed;
+	width:100%;
+	height:75px;
+	background-color: RGBA(27,49,67,.8);
+	z-index: 9999;
+}
+
+ul {
+	display:inline-block;
+	margin:0;
+}
+
+li {
+	float:left;
+	list-style-type:none;
+	margin:10px;
+	font-size:2rem;
+	padding-top:20px;
 }
 
 header {
-	background-image:url();
+	padding-top:0px;
+	height:900px;
+	width:100%;
+	overflow:auto;
+/*	background-image:url('img/binary.jpg');*/
+	background-size:cover;
+	
+}
+
+.intro {
+	position:relative;
+	background-image:url('img/textBackground.png');
+	background-size:100% 500px;
+	background-position:10%;
+	padding:80px 0px 350px 0px;
+	margin-top:100px;
+}
+
+
+.intro h1 {
+	position:absolute;
+	top:190px;
+	left:22%;
+	transform:rotate(-7.5deg)skewX(-20deg);
+	font-size:3.3rem;
+	margin:0;
+}
+
+.intro h3 {
+	position:absolute;
+	top:225px;
+	left:30%;
+	transform:rotate(-7.5deg)skewX(-20deg);
+	font-size:3.2rem;
+	margin-top:10px;
+}
+
+.moreInfo {
+	position:absolute;
+	top:300px;
+	left:45%;
+	transform:rotate(-7.5deg)skewX(-20deg);
+	background-color:transparent;
+	border: 3px white solid;
+	padding-top:5px;
+	padding-left:20px;
+	padding-right:20px;
+	font-size:2rem;
+	color:#FC630E;
+}
+
+.moreInfo:hover {
+	color:white;
+	padding-top:5px;
+	padding-left:25px;
+	padding-bottom:5px;
+	padding-right:25px;
+	font-size:2.3rem;
+	transition:ease 0.5s;
+	border: #FC630E solid 3px;
+}
+
+i {
+	border: solid #840CE1;
+	border-width: 0 10px 10px 0;
+	display: inline-block;
+	padding: 10px;
+	position:absolute;
+	top:85%;
+	left:49%;
+	
+}
+
+.down {
+	transform: rotate(45deg);
+	-webkit-transform: rotate(45deg);
+}
+
+.seePortfolio {
+	color:#FC630E;
+	position:absolute;
+	top:90%;
+	left:47%;
 	
 }
+
 .circleImg {
 	background-image: url('https://www.thebalance.com/thmb/xdaHH5q9Wd4kpvlnJxj6WVbayu0=/400x0/portfolio-175431947-5ad433371f4e13003890fcdc.jpg');
 	  background-color: #cccccc;
-	  height: 100px; 
-	  width:100px;
```

Comment:
Height and width defined by another identically-named circleImg class, starting on line 643.